Tag content updates (#921)
hashicorp/design-system: website/docs/components/tag/partials/guidelines/guidelines.md

## Usage

### When to use

Use tags to indicate an object's categorization, i.e., for filtering.

### When not to use

- For static metadata, status, or to display a new feature, use [Badge](/components/badge/overview).
- For collection enumeration or version number, use [BadgeCount](/components/badge-count/overview).

### Layout

> - The tag component can be seen in several different layouts, but we commonly see them in a list or a group.

Reviewer: ```suggestion
- The tag component can be used in several different layouts, but we commonly see them in a list or a group.
```

- Tags will fill the parent container and then wrap, as necessary.

#### List

![Layout list example](/assets/components/tag/tag-layout-list.png)

#### Group

![Layout group example](/assets/components/tag/tag-layout-group.png)

---

### Spacing

> As a best practice, we recommend a vertical spacing of 12px and horizontal spacing of 8px between tags.

Reviewer: ```suggestion
As a best practice, use a vertical spacing of 12px and horizontal spacing of 8px between tags.
```
This is a bit more actionable

![Spacing between tags](/assets/components/tag/tag-spacing.png)

---

### Leading vs. Trailing Icon

> The leading dismiss icon allows mouse users to clear multiple tags quicker, because they do not need to move their mouse, especially when tags have a more extended width.

Reviewer: ```suggestion
The leading dismiss icon allows mouse users to clear multiple tags quicker because they do not need to move their mouse, especially when tags have a more extended width.
```

> ## Usage

Reviewer 2: ```suggestion
## Usage

```

## Usage

### When to use

Use tags to indicate an object's categorization, i.e., for filtering.

### When not to use

> - For static metadata, status, or to display a new feature, use [Badge](/components/badge/overview)

Reviewer 3: - For static metadata, status, or to display a new feature, use [Badge](/components/badge/overview).

> - For static metadata, status, or to display a new feature, use [Badge](/components/badge/overview)

Reviewer 3: ```suggestion
- For static metadata, status, or to display a new feature, use [Badge] (https://github.com/components/badge/overview).
```

> - For static metadata, status, or to display a new feature, use [Badge](/components/badge/overview)

Reviewer 3: ```suggestion
- For static metadata, status, or to display a new feature, use [Badge](/components/badge/overview).
```

> - For collection enumeration or version number, use [BadgeCount](/components/badge-count/overview)

Reviewer 3: - For collection enumeration or version number, use [BadgeCount](/components/badge-count/overview).

### Layout

> - The Tag component can be seen in several different layouts, but we commonly see them in a list or a gorup.

Reviewer 3: The tag component can be seen in several different layouts, but we commonly see them in a list or a group.

> - The Tag component can be seen in several different layouts, but we commonly see them in a list or a gorup.

Reviewer 3: ```suggestion
- The tag component can be seen in several different layouts, but we commonly see them in a list or a group.
```

> - Tags will fill the parent container and then wrap, as necessary

Reviewer 3: ```suggestion
- Tags will fill the parent container and then wrap, as necessary.
```

#### List

![Layout list example](/assets/components/tag/tag-layout-list.png)

#### Group

![Layout group example](/assets/components/tag/tag-layout-group.png)

---

### Spacing

> As a best practice, we recommend a vertical spacing of 12px and horiztonal spacing of 8px between tags.

Reviewer 3: As a best practice, we recommend a vertical spacing of 12px and horizontal spacing of 8px between tags.

> As a best practice, we recommend a vertical spacing of 12px and horiztonal spacing of 8px between tags.

Reviewer 3: ```suggestion
As a best practice, we recommend a vertical spacing of 12px and horizontal spacing of 8px between tags.
```
